internal/services/compute/shared_schema.go

```diff
@@ -293,7 +293,7 @@ func planSchema() *pluginsdk.Schema {
 }
 
 func expandPlan(input []interface{}) *compute.Plan {
-	if len(input) == 0 {
+	if input == nil || len(input) == 0 {
```

Comment:
actually, I think this should be
```suggestion
	if len(input) == 0 || input[0] == nil {
```
?

Reply:
Yep, thank you! Lint wasn't happy with me as well :)